generation/new_client_hermetic_build/README.md

```diff
@@ -0,0 +1,160 @@
+# New client generation (GitHub Action)
+This new generation workflow enables generation of new libraries by
+ - appending a new library to our [generation_config.yaml](https://github.com/googleapis/google-cloud-java/blob/c7429c0eec419c01d4e2fe14d063b9335efb810b/generation_config.yaml).
+ - running the hermetic build scripts docker image and
+ generate the newly added library.
+ - create a PR with the changes.
+
+
+## Components
+### `new_library_hermetic_build/new-client.py`
+This script takes 10 arguments that map to items in the newly added library that
+goes in `generation_config.yaml`. A new entry will be added to `libraries` with
+the necessary generation configuration.
+
+### `.github/workflows/generate_new_client_hermetic_build.yaml`
+This workflow orchestrates the `new-client.py` script and also uses our docker
+image
+([gcr.io/cloud-devrel-public-resources/java-library-generation](https://pantheon.corp.google.com/gcr/images/cloud-devrel-public-resources/global/java-library-generation))
+to generate a new library. It also creates the pull request.
+
+
+## Execute the Github Action
+
+You in order to run the
+[Github Action](https://github.com/googleapis/google-cloud-java/actions/workflows/generate_new_client_hermetic_build.yaml)
+, you need to specify a few parameters.
+These parameters will be available in the Cloud Drop link (a YAML file) included in the buganizer request.
+The example in this README uses AlloyDB's [Cloud Drop](https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/alloydb_v1.yaml) file as an example.
+
+### API short name (`api_shortname`)
+
+For convenience of the subsequent commands, define a variable for API short name.
+This value will be used by default to generate the following:
+* `--distribution-name`
+* `--destination-name`
```

Comment:
I `--destination-name` is no longer needed.

Reply:
Indeed, removed. Thanks